emde/plugins/org.polarsys.kitalpha.emde.model/generated/org/polarsys/kitalpha/emde/model/util/EmdeResourceImpl.java

```diff
@@ -45,24 +45,6 @@
  * @generated
  */
 public class EmdeResourceImpl extends XMIResourceImpl {
```

Comment:
Please check the missing methods in the new version

Reply:
Indeed there is problem here this code is missing.
Thanks for noticing.